0036-spsp-pull-payments/0036-spsp-pull-payments.md

```diff
@@ -0,0 +1,133 @@
+---
+title: SPSP Pull Payments
+draft: 1
+---
+# SPSP Pull Payments
+
+## Preface
+
+This document describes how to conduct pull payments via [STREAM](../0029-stream/0029-stream.md) once the payment details have been exchanged via the [Simple Payment Setup Protocol](../0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md) (SPSP).
+
+## Introduction
+
+### Motivation
+
+The Simple Payment Setup Protocol (SPSP) only descirbes how the details required for a STREAM connection are exchanged and how a simple push payment is made. However, it lacks a detailed explaination of how a pull payment is conducted using this connection. 
```

Comment:
`descirbes` should be `describes`.

Reply:
Thanks, I'll add the fix to the next commit.